### DISCUSSION TITLE
300 celcius?

### DISCUSSION THREAD

**Discussion by Swe-Shogoki** [Jyers/Marlin#1964] [Ideas]
Just started whit this thing and try to make a firmware based on this one [E3V2-ManualMesh-5x5-v4.2.2-v2.0.1.bin]  Need it to be on extruder temp 315 and not the 275 thats is oem in this firmware. If i use the source code itj just all of the things in this firmware OF. any one that have one for higher temp on the extruder?

**Comment by Swe-Shogoki** [comment]
Solved it Now got 300c and 5x Mesh. Have the firmware if some one whant it

**Reply by CHARGERFORCE** [reply]
What did you end up doing?
